plugins/search/src/components/SearchBar/SearchBar.tsx

```diff
@@ -36,13 +36,14 @@ import {
   SearchContextProvider,
   useSearch,
   useSearchContextCheck,
+  TrackSearch,
 } from '@backstage/plugin-search-react';
-import { TrackSearch } from '../SearchTracker';
 
 /**
  * Props for {@link SearchBarBase}.
  *
  * @public
+ * @deprecated Moved to `@backstage/plugin-search-react`.
```

Comment:
These messages are often displayed in IDEs -- phrasing it from an app integrator / plugin developer's perspective (e.g. making it actionable) is a nice touch!  E.g.

```
@deprecated Import from `@backstage/plugin-search-react` instead
```

(Comment applies throughout PR)